WhoIsThat/WhoIsThat/App.xaml.cs

```diff
@@ -19,14 +20,15 @@ public App()
             MainPage = new NavigationPage(new MainPage(new ViewModels.LoginViewModel()));
         }
 
-        protected override void OnStart()
+        protected override async void OnStart()
         {
+            var restService = new RestService();
             // Handle when your app starts
             if (Application.Current.Properties.ContainsKey("UserRegistered"))
             {
                 if (Application.Current.Properties["UserRegistered"].Equals(true))
                 {
-                    MainPage = new NavigationPage(new HomePage(new ViewModels.HomeViewModel()));
+                    MainPage = new NavigationPage(new HomePage(new ViewModels.HomeViewModel(await restService.GetUserById(Convert.ToInt32(Application.Current.Properties["UserId"].ToString())))));
```

Comment:
Thats a little bit too long of a method call, isnt it?:D

Reply:
And why do you first convert to string, then back to int?

Reply:
Now I can't remember exactly, but there was a reason, I think it's not exactly a string of dictionary. I will double - check that though.

Yeah, you are right, it's a bit long. It's just that it wasn't me who came up with this solution, so I wanted to check if it's really working. Will work on beauty here, and ask for your review again.

Will keep that PR open for couple of days.